db/migrate/20190904144157_add_state_to_backend_apis.rb

```diff
@@ -0,0 +1,8 @@
+class AddStateToBackendApis < ActiveRecord::Migration
+  disable_ddl_transaction! if System::Database.postgres?
+
+  def change
+    add_column :backend_apis, :state, :string
```

Comment:
We could add a default here I think. On SaaS:

```
irb(main):001:0> BackendApi.count
=> 37183
```

It will be blocking but reasonably, and in fact it might not block at all, there are very few requests that modifies the private_endpoint